.gitignore

```diff
@@ -19,3 +19,4 @@ tests/coverage.info
 *.jar
 *.iml
 .idea/*
+/target/
```

Comment:
Quoting from https://git-scm.com/docs/gitignore
'A leading slash matches the beginning of the pathname. For example, "/*.c" matches "cat-file.c" but not "mozilla-sha1/sha1.c".'
I want to match only the top level target directory created by maven